nesc/whip6/apps/AsyncReadDemo/AsyncReadDemoPrv.nc

```diff
@@ -0,0 +1,62 @@
+/**
+ * whip6: Warsaw High-performance IPv6.
+ *
+ * Copyright (c) 2017 Uniwersytet Warszawski
+ * All rights reserved.
+ */
+
+module AsyncReadDemoPrv {
+    uses interface Boot;
+    uses interface Led as Green;
+    uses interface Led as Orange;
+    uses interface Led as Red;
+    uses interface Led as Yellow;
+    uses interface Timer<TMilli, uint32_t>;
+    uses interface ReadNow<uint8_t>;
+}
+
+implementation {
+    uint8_t locVal;
```

Comment:
Next time, prefix a module's local variables with "m_" (e.g., "m_locVal"). In this way, it is easier to navigate in the code.